template.tpl

```diff
@@ -367,6 +408,7 @@ var setInWindow = require('setInWindow');
 var callInWindow = require('callInWindow');
 var createQueue = require('createQueue');
 var makeTableMap = require('makeTableMap');
+var JSON = require('JSON');
```

Comment:
Ah, no wonder there's a `require` for the native object: `___SANDBOXED_JS_FOR_WEB_TEMPLATE___`.